docs/data-sources/team.md

```diff
@@ -8,14 +8,25 @@ Buildkite documentation: https://buildkite.com/docs/pipelines/permissions
 ## Example Usage
 
 ```hcl
-data "buildkite_team" "my_team" {
-  slug = "my_team"
+data "buildkite_team" "my_team_data" {
+    id = "<team id>"
 }
 ```
 
+The above will automatically fill the ID attribute with that of the team created by the resource, then fill the other
+attributes of that team.
+
+The datasource can also reference an imported Team. For more on using Import check
+[here](https://github.com/buildkite/terraform-provider-buildkite/blob/main/docs/resources/team.md#import).
+
```

Comment:
Last thing: I don't think this section fits either. Importing and datasource are basically 2 sides of the same coin. You wouldn't import a resource and then use a datasource for the same resource. I think this entire section could probably be removed